packages/client/src/components/Notification/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Notifications.css';
+import UncontrolledAlert from './UncontrolledAlert';
+
+/**
+ * Copyright (c) 2016-Present Eddy Hernandez, Chris Burrell, Evan Sharp
+ * package: react-strap
+ */
+
+function Notification (props) {
+    if (props.id === 'sign-in') {
```

Comment:
`id` is canonically reserved for giving a unique identifier to an html element, we shouldn't use to signify a category/type since we might have multiple notifications of the same type at once on the screen.

In general it seems simpler for the consuming components to just use `UncontrolledAlert` directly

Reply:
Done and styled.